testing/examples/basic/cloudbuild.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Tests that verify the basic example of rules_docker vs Dockerfile.
+
+steps:
+  # Confirm bazel and docker are installed.
+  - name: "gcr.io/asci-toolchain/nosla-ubuntu16_04-bazel-docker-gcloud"
+    entrypoint: "sh"
+    args: ["-c", "bazel version && docker version"]
+    id: "versions"
+    waitFor: ["-"]
+
+  # Test the basic example.
+  - name: "gcr.io/asci-toolchain/nosla-ubuntu16_04-bazel-docker-gcloud"
+    entrypoint: "sh"
+    args: ["-c", "cd testing/examples && bazel test //basic:all"]
+    waitFor: ["versions"]
```

Comment:
use 'dir:' to set the directory to testing/examples (Example: https://github.com/bazelbuild/bazel-toolchains/blob/master/tests/rbe_repo/examples/rbe_custom.yaml#L8)
That way you dont need an entrypoint and the only args you should pass are 
 - test 
 - //basic:all

Reply:
Done.